From f91a8fbf3ac94015c0d4d53fa0560ed01061c46f Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Tue, 13 Feb 2018 23:06:57 +0530
Subject: [PATCH 0265/1851] dt: bindings: media: xilinx: Add rev 3.0 to
 csi2rxss compatible string

commit 90504618692c0a3acd4c2fd7271ee4f63da6ba12 from
https://github.com/Xilinx/linux-xlnx.git

Add 3.0 version number to the xlnx,csi2rxss compatible string.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt         | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
index 6ac6a0847d2b..3d948eaad402 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
@@ -14,7 +14,8 @@ For more details, please refer to PG232 MIPI CSI-2 Receiver Subsystem v2.0
 
 Required properties:
 
-- compatible: Must contain "xlnx,mipi-csi2-rx-subsystem-2.0"
+- compatible: Must contain "xlnx,mipi-csi2-rx-subsystem-2.0" or
+  "xlnx,mipi-csi2-rx-subsystem-3.0"
 
 - reg: Physical base address and length of the registers set for the device.
 
-- 
2.31.1

